src/main/scala/com/azavea/modellab/Parser.scala

```diff
@@ -45,12 +46,20 @@ class Parser(layerRegistry: LayerRegistry, layerReader: FilteringLayerReader[Lay
   implicit class withJsonMethods(json: JsValue) {
     val fields = json.asJsObject.fields
     def get[T: JsonReader](name: String) = fields(name).convertTo[T]
-    def param[T: JsonReader](name: String) = 
+    def param[T: JsonReader](name: String): T = 
       fields("parameters")
         .asJsObject
         .fields(name)
         .convertTo[T]
 
+    def optParam[T: JsonReader](name: String): Option[T] = 
+      Try(fields("parameters")
+        .asJsObject
+        .fields(name)
+        .convertTo[T]
+      ).toOption
```

Comment:
Not really certain how I feel about the above method. @echeipesh do you know of a cleaner way than this?